.changeset/calm-eels-hope.md

```diff
@@ -0,0 +1,8 @@
+---
+'@graphql-mesh/transport-common': patch
+'@graphql-tools/executor-http': patch
+'@graphql-mesh/fusion-runtime': patch
+'@graphql-hive/gateway-runtime': patch
+---
+
+Fix Retry / Timeout combination
```

Comment:
can you please improve this? if we cut a release for packages, we should include better description of the changes.

Reply:
I added some more information